modules/tinymce/src/themes/silver/main/ts/ui/menus/contextmenu/SilverContextMenu.ts

```diff
@@ -119,15 +120,16 @@ export const isTriggeredByKeyboard = (editor: Editor, e: PointerEvent) =>
 const getSelectedElement = (editor: Editor, e: PointerEvent) =>
   isTriggeredByKeyboard(editor, e) ? editor.selection.getStart(true) : e.target as Element;
 
-const shouldUseNodeAnchor = (editor: Editor, e: PointerEvent) => {
+const getAnchorType = (editor: Editor, e: PointerEvent): ContextMenuAnchorType => {
   const selector = Settings.getAvoidOverlapSelector(editor);
-  if (isTriggeredByKeyboard(editor, e)) {
-    return true;
-  } else if (selector) {
+  if (selector) {
     const target = getSelectedElement(editor, e);
-    return SelectorExists.closest(SugarElement.fromDom(target), selector);
+    const selectorExists = SelectorExists.closest(SugarElement.fromDom(target), selector);
+    return selectorExists ? 'node' : 'point';
```

Comment:
This fallback is incorrect. It needs to continue on to the logic below instead when the selector doesn't exist.

Reply:
Changed.